cdap-app-fabric/src/main/java/io/cdap/cdap/support/status/SupportBundleStatus.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.support.status;
+
+/**
+ * Status when generating Support bundle.
+ */
+public class SupportBundleStatus {
+  // UUID of the bundle status object describes
+  private String bundleId;
```

Comment:
Everything can be final, is not it? 
Also it's better to put comments into javadocs